### DISCUSSION TITLE
External Load Balancer IP update

### DISCUSSION THREAD

**Discussion by mgup73** [strimzi/strimzi-kafka-operator#8909] [General]
Hi Team,

I was working on adding the external Load balancer Source ranges in the Kafka cluster setup for bootstraps/brokers and my new IPs are not getting updated cluster. Any idea? Do I need to re-install to add the new set of IPs in source ranges?

**Comment by scholzj** [comment]
All Strimzi does is set the source ranges on the service. So you can check if they were updated or not by doing `kubectl get service -o yaml`. But the rest of it is up to your cloud provider. So you would need to check there if it can be updated or set only once etc.
